src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -349,13 +362,22 @@
           community
           pending?
           {:on-category-layout              (partial add-category-height categories-heights)
+           :collapsed?                      collapsed?
            :on-first-channel-height-changed
            ;; Here we set the height of the component and we filter out the
            ;; categories, as some might have been removed
            (fn [height categories]
              (swap! categories-heights select-keys categories)
              (reset! first-channel-height height))}]]))))
 
+(defn community-card-page-view
+  []
+  (fn [id]
```

Comment:
This component doesn't need to be a form-2 component, so it seems safe to remove the anonymous function.